invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/AccessRequestForm.js

```diff
@@ -76,11 +78,23 @@ export class AccessRequestForm extends Component {
     } catch (error) {
       if (error === "UNMOUNTED") return;
 
+      const errorMessage = error?.response?.data?.message || error?.message;
+
       this.setState({
-        error: error?.response?.data?.message || error?.message,
+        error: errorMessage,
         loading: false,
       });
       console.error(error);
+
+      // Retry the POST request if the request failed on CORS validation.
+      const corsError =
```

Comment:
Not the most elegant solution but for now we settled on making it work.

We know what the "issue" is, but we don't have a general solution for this.

Therefore, I created a new issue to have a deeper look into this https://github.com/inveniosoftware/invenio-app-rdm/issues/2413